docker/Dockerfile

```diff
@@ -1,51 +1,97 @@
 ARG PYTHON_VER
-################################ Stage: base
-# Install all OS package upgrades and dependencies once from which we will base other containers
+ARG PYUWSGI_VER
+
```

Comment:
Thoughts on adding EXPOSE for ports we are running on? Some systems will pick up those ports and perform port mapping automatically for you, and it's nice from a documentation perspective for the expected ports to be called out in the image

Reply:
Maybe? I think that'd just be 80 for the dev/final-dev images and 443 for the final image though.